docs/adr/0009-publish-chart-to-rancher-charts.md

```diff
@@ -0,0 +1,95 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [9. Helm chart repository](#9-helm-chart-repository)
+  - [Context](#context)
+  - [Proposed alternatives](#proposed-alternatives)
+  - [Consequences](#consequences)
+  - [Decision](#decision)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# 9. Helm chart repository
+
+* Status: proposed
+* Date: 2024-03-11
+* Authors: @salasberryfin
+* Deciders: @richardcase @alexander-demicev @Danil-Grigorev @mjura @furkatgofurov7
+
+## Context
+
+For Rancher Turtles to become GA, the Helm chart must be made available through the Rancher Marketplace. There are two alternatives to serve the chart:
```

Comment:
I am not sure if this statement is accurate (usage of MUST), at the end of the day, going with option 2 means the chart NOT being available through the Rancher Marketplace, right?